features/support/world.ts

```diff
@@ -63,7 +63,7 @@ export class World {
       const cli = new Cli({
         argv: args,
         cwd,
-        stdout,
+        stdout: (stdout as unknown) as WriteStream,
```

Comment:
Why is this needed? Is stdout not a WriteStream?

Reply:
We're using a `PassThrough` here, which doesn't extend either the fs or tty `WriteStream` although it has all the methods we use.  Unfortunately the closest shared interface between all three is `Writable` but that that is missing stuff and would give us compilation issues.

We can avoid this by adding `PassThrough` to the `IFormatterStream` interface - I have done that for now, let me know what you think.